content/60.methods.md

```diff
@@ -123,24 +130,50 @@ Because in this implementation of Manubot, most collaborators were writing and e
+<!-- To Do: use comments on #17 to illustrate engagement in issue number 17 and diversity of interest (Casey independently suggested a word cloud)-->
+Because the GitHub issues and comment systems are relatively similar to other web common web activities, we found that authors were able to learn these tools fairly quickly.
+Similarly, the gitter chat also presented a low barrier to entry.
+While only a fraction of potential contributors ended up contributing to the development of the manuscript, many of these contributors remained engaged over the course of a full year (Figure with dots).
 
-#### Data Analysis and Integration
+<!--To Do: Add dot plot of unique contributors & commits over time-->
 
-<!--To Do: Flow chart of data integration? We could have a summary figure showing all of the external data sources that are integrated into the manuscript. We have icons for MSSM reviews, JHU data, etc. at the top. That flows into a GitHub repo, which also takes input from all the contributors. Then the output of the repo is the manuscript and other CI artifacts.-->
+In order to make the project accessible to individuals from a number of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
+We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
+Additionally, the framework for evaluating literature was converted into issue templates to simplify review of new articles.
+Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.
+<!-- To Do: Number of paper issue and number of unique contributors?-->
+
+Currently, three manuscripts associated with this project have been submitted to _mSystems_, with one accepted for publication [@arxiv:2102.02250] and two under revision [@arxiv:2103.02723; @arxiv:2102.01521].
+An additional four manuscripts are in preparation.
+All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they will continue to be updated as more information becomes available.
```

Comment:
```suggestion
All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
```

"Will" seemed a bit strong to me given how much work it takes to get the first versions out.